README.md

```diff
@@ -223,6 +223,9 @@ You can install ComfyUI in Apple Mac silicon (M1 or M2) with any recent macOS ve
 #### DirectML (AMD Cards on Windows)
 
 ```pip install torch-directml``` Then you can launch ComfyUI with: ```python main.py --directml```
+#### ZLUDA (AMD Cards on Windows)
+If your AMD GPU are 7000 series or a newer one and you have problems using direcml,ZLUDA would be more effective when generating content.
+ To know the detailed usage, follow the instructions [here](https://github.com/vosen/ZLUDA)
```

Comment:
Suggested change:

If you have a 7000 series or newer AMD GPU and are experiencing issues with DirectML, ZLUDA may be an option. For more information, see the [official ZLUDA repository](https://github.com/vosen/ZLUDA).